docs/Reply.md

```diff
@@ -177,8 +177,12 @@ fastify.get('/', function (request, reply) {
   reply.send(httpErrors.Gone())
 })
 ```
+By default Fastify will log the error depending on the `statusCode`:
+- for `400 <= statusCode < 500` the error will be logged with level `info`
+- for `statusCode >= 500` the error will be logged with level `error`
```

Comment:
What's happening when statusCode < 400?

Reply:
It is mapped to 500 by `handleError` before calling the `errorHandler`.

https://github.com/fastify/fastify/blob/39225010c4a7681bcbd4d137463449677908ce6b/lib/reply.js#L284-L299

Should it be made clearer?

Reply:
Yeah. Because if an error with statusCode 302 and headers property with "Location" is thrown, the output is a 500 with Location header set (and the browser will not perform a redirection)

Reply:
Yeah, so the redirection is not performed even if the `statusCode` is set to 302.

So, I'd like to report this behavior in our doc. Maybe a 'All error with a `statusCode` property lower then 400 are treated as 500 ones' is sufficient.

Reply:
It is written few line above
https://github.com/fastify/fastify/pull/1135/files/fc6d50a2145b186c467e13be3ea2bcf5f3b9baf3#diff-31032738bcacc769cecf4d9f6989d581L171

I can maybe move my addition right below those lines

Reply:
I described both the replacement and the log when status < 400